### DISCUSSION TITLE
Error

### DISCUSSION THREAD

**Discussion by GoldGamerPT** [supabase/supabase#20989] [Questions]
![image](https://github.com/supabase/supabase/assets/75499273/f18f272b-843e-4d34-8715-498e1b92d7a3)

i don't know what error this is can someone help me please? If you need I can provide more information, i'm making a quiz app and when i host the game it freezes and this happens!

**Comment by GaryAustin1** [comment]
Possibly you don't meet RLS for either select or insert?  Try with RLS off and see if it works.

**Reply by GoldGamerPT** [reply]
how can do that? sorry to ask so many questions

**Reply by GaryAustin1** [reply]
![image](https://github.com/supabase/supabase/assets/54564956/438084e0-e83a-4cc2-9f21-3639c6346a11)
